src/clr-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -57,7 +57,7 @@ export class ClrTabLink {
     public ifActiveService: IfActiveService,
     @Inject(IF_ACTIVE_ID) private id: number,
     private ariaService: AriaService,
-    private el: ElementRef,
+    public el: ElementRef,
```

Comment:
Does it need to be public?

Reply:
Yes, unfortunately I need to expose it so the `Tabs` component can get a reference to its element ref to build the list of focusable items.

Reply:
No, I have a better way to accomplish this. Will have a commit up soon.